examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,16 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+kws_psd = dict(tmin=tmin, tmax=tmax, fmin=fmin, fmax=fmax, n_jobs=1)
+psds_wel, freqs_wel = psd_welch(raw, n_fft=n_fft, **kws_psd)
+psds_mt, freqs_mt = psd_multitaper(raw, low_bias=True, **kws_psd)
```

Comment:
I like the fact that the additions are pretty concise. But I'm worried users will be confused if they are shown two different ways of doing the same thing (psd_welch). I would just put the one line for `psd_multitaper` and a simple plot (no subplots). It's may come as a surprise but even things like `zip` etc., can turn away people if it's in the examples